### ISSUE TITLE
Recurring Events "Ends" dropdown exists on the page at -1,000px

### ISSUE THREAD

**Issue by srancour** [department-of-veterans-affairs/va.gov-cms#19162]
## Statuses
**[2025-01-9] [Fran]** Since this is a defect, and we still own Events, we have refined it

## Describe the defect
In Drupal, when an event is made to be recurring there is a dropdown for "Ends" after the "Repeat every" "Repeat" type dropdown in the DOM. On the page, the "Ends" dropdown sits at -1,000px due to a `left: -1,000px;` on the `.form-item--field-datetime-range-timezone-0-repeat-end`css class. This makes it so anyone on a screen smaller than 2915px can't see it, but it still exists and can be tabbed to with keyboard and a screen reader user still encounters it.

## To Reproduce
1. Edit an event that's recurring like [Music Therapy Jam Session](https://staging.cms.va.gov/central-iowa-health-care/events/53044)
2. Tab though options after "Repeat every" input and dropdown
3. Either see focus onto "Ends" dropdown if screen size is at least 2915px wide, or see focus move onto nothing and then back on second tab.

## Notes 2025-01-09
- See this comment below: https://github.com/department-of-veterans-affairs/va.gov-cms/issues/19162#issuecomment-2339302008

## AC / Expected behavior
- [ ] The value for Ends should be permanently set to On Date
- [ ] When "Make recurring" is selected on the event form, the Ends field should not be readable by screen readers or visible on screen

## Screenshots
![Screenshot of part of the dropdown visible to the left of the screen with a green focus on it](https://github.com/user-attachments/assets/d7b9b788-60f9-4032-95a0-7055ede398d4)
![Screenshot of the dropdown opened and on the 'After' option showing that the option to input text goes in under the 'On these days' option](https://github.com/user-attachments/assets/077c62f1-bba9-4b11-af63-8220bb3ec475)

 ## Additional context
This seems to have been here for 2 years since the [Recurring Events feature](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/6057) was completed at pull #7981. It looks like it came from a [request to 'Remove "Never/After/On date" dropdown'](https://github.com/department-of-veterans-affairs/va.gov-cms/pull/7307#issuecomment-998203897) in a comment on #7037 by @kevwalsh from 2021.

## Desktop (please complete the following information if relevant, or delete)
 - Window width of at least 2915px to visually see dropdown

**Comment by jilladams** [comment]
@laflannery despite not knowing the destiny of Events and whether we'll update them and when / where: Steve found an a11y issue, if you would like to put a defect level on it.

**Comment by laflannery** [comment]
@EWashb and @jilladams This is a regression - [This was resolved 2 years ago](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/10324#issuecomment-1256217120) (almost to the day weirdly enough)

My best guess is that it might have had to do with the recent upgrade?

**Comment by srancour** [comment]
@laflannery if you look at the class in the code, it hasn’t changed since June of 2024 when the #7981 push happened. I also found this on prod on Friday before the update to Drupal went to prod, so I don’t think it had to do with this Drupal update. From what I could find in the linking tickets, there was a similar issue where the “All Day” checkbox was also at -1,000px, but I couldn’t find this as a bug previously found or fixed.

**Comment by jilladams** [comment]
I got a little lost in the link sauce there, but stepping through it: 
* #10131 was filed and closed as resolved by #10324
* #10324 had one PR for a change that just removed a style on the All Day checkbox label: https://github.com/department-of-veterans-affairs/va.gov-cms/pull/10669/files. That change resulted in the All Day checkbox being `dispay:none` rather than pushed 1000px offscreen to the left, but Christia's PR does still show it in the DOM.
  * Steve Wirt did ask:  why not remove it from the form in docroot/modules/custom/va_gov_events/src/EventSubscriber/EntityEventSubscriber.php : alterEventNodeForm() 
  * Christia noted: the All Day field is added by Javascript, so can't be removed by form alter.

I'm not finding a comment anywhere that we were able to verify the fix on #10131, so it was prob lost in the shuffle of no Steve response. 

If / when we pick this up, we will want to think through how we might be able to modify the JS to remove the All Day field as well (rather than display: none / having it present in the dom) as well as the "Ends" field, based on that PR / comments.

**Comment by FranECross** [comment]
@jilladams I updated the AC and added a Notes section with today's date on it, based on your comment above. Will you review and see if the AC is correct based on your understanding? Thanks!

**Comment by jilladams** [comment]
Took a look and updated the AC. Thanks!

**Comment by jilladams** [comment]
Moving back to Ready to Deploy since CMS is not regularly deploying at the mo.

**Comment by dsasser** [comment]
Verified on Prod:

<img width="1340" alt="Image" src="https://github.com/user-attachments/assets/6280ccf1-04f6-4497-82e7-ec2cf5c26341" />
<img width="1435" alt="Image" src="https://github.com/user-attachments/assets/d5831440-7bae-4d11-9c10-885ab5d1320e" />
